README.md

```diff
@@ -28,14 +26,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories which needn't necessarily be merged in October.
+
+    * 1 `Difficulty: Challenging` PR or 2 `Difficulty: Medium` PRs or 5 `Difficulty: Starter` PRs: limited edition t-shirt and stickers
```

Comment:
Please remove the ` symbols here as they are used for inline code. You can see how it affects the Jekyll build by [looking at the deploy preview here](https://deploy-preview-86--hacktoberfest-swag-list.netlify.com/)